devel/hs-transformers/Portfile

```diff
@@ -0,0 +1,20 @@
+PortSystem          1.0
+PortGroup           haskell 1.0
+
+haskell.setup       transformers 0.4.3.0
+name                hs-transformers
+checksums           rmd160  17d83a518f96280f35b8f9c0928fcf1235f45d85 \
+                    sha256  b3d0a797e815ca50d411e20c02f781efe7751308007d880af7f0b5c4365c3a9d
+
+license             GPL-3+
```

Comment:
According to http://hackage.haskell.org/package/transformers, this should be BSD.